### PR TITLE
Update bits/architecture and better error handling

### PR DESCRIPTION
**Pull request by tupui**

### CODE REVIEW

File: rbc/targetinfo.py [hunk 1]
```diff
@@ -282,8 +282,23 @@ def bits(self):
...
-        return dict(x86_64=64, nvptx64=64,
-                    x86=32, nvptx=32, arm64=64)[self.arch]
+        bits = dict(
+            x86_64=64, nvptx64=64, arm64=64, aarch64=64,
+            x86=32, nvptx=32,
+        ).get(self.arch)
+        if bits is None:
+            if '64' in self.arch:
+                warnings.warn(
+                    f"Unknown {self.arch} architecture: inferring 64bits."
```
Comment (pearu) [inline rbc/targetinfo.py:292]:
```suggestion
                    f"Unknown {self.arch} architecture: inferring 64 bits."
```

**Review by pearu — APPROVED** [review]
I have minor nit and LGTM! Thanks, @tupui!

I wonder why CI tests have not run. Any ideas?

**Comment by tupui** [comment]
@pearu ah no, the branch changed but not the config of the CI. Shall I update this here?

**Comment by tupui** [comment]
(I will fix the notebook with ibis in another PR)

**Review by pearu — APPROVED** [review]
LGTM! Thanks, @tupui!

**Pull request merged**